content/modules/enterprise-capabilities.md

```diff
@@ -17,13 +17,13 @@ The following table shows which modules are supported by Redis Enterprise Softwa
 
 | Module | Redis Enterprise<br/>Software | Redis Enterprise<br/>Cloud |
 |--------|:-------------------------:|:-----------------------:|
-| [RedisAI]({{<relref "/modules/redisai">}})       | Yes | No |
-| [RedisBloom]({{<relref "/modules/redisbloom">}}) | Yes | Yes |
-| [RedisGears]({{<relref "/modules/redisgears">}}) | Yes | No |
-| [RedisGraph]({{<relref "/modules/redisgraph">}}) | Yes | Yes |
-| [RedisJSON]({{<relref "/modules/redisjson">}})   | Yes | Yes |
 | [RediSearch]({{<relref "/modules/redisearch">}}) | Yes | Yes |
+| [RedisJSON]({{<relref "/modules/redisjson">}})   | Yes | Yes |
+| [RedisGraph]({{<relref "/modules/redisgraph">}}) | Yes | Yes |
 | [RedisTimeSeries]({{<relref "/modules/redistimeseries">}}) | Yes | Yes |
+| [RedisGears]({{<relref "/modules/redisgears">}}) | Yes | No |
+| [RedisBloom]({{<relref "/modules/redisbloom">}}) | Yes | Yes |
```

Comment:
The order of RedisGears and RedisBloom do not match the ToC.
```suggestion
| [RedisBloom]({{<relref "/modules/redisbloom">}}) | Yes | Yes |
| [RedisGears]({{<relref "/modules/redisgears">}}) | Yes | No |
```

Reply:
Fixed